### PR TITLE
ANSI colors

### PR DESCRIPTION
**Pull request by Tokimon**
I am currently working on Windows and found it sad to see the pretty Nyan cat converted to gray colors, so  I made an alternative simplified version of the rainbow colors in ANSI colors which is supported by all terminals.

The ANSI colors are activated by adding an extra argument to the Nyan object, which is done by adding `--ansi` to the CLI call.

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
is it all windows terminals or is it just cmd.exe or powershell ?

**Comment by Tokimon** [comment]
this is cmd.exe (and ConEmu). Just really catering to all the poor developers that are stuck with windows really. It is still using the full color spectrum if `ansi` is not given. so it is really just a feature to please developers on windows.

**Comment by calvinmetcalf** [comment]
I was more wondering if we could guess they needed it automatically

**Comment by Tokimon** [comment]
oh... hm... I can try to look into it, but it might be tricky :-/

**Comment by Tokimon** [comment]
Ok guess it wasn't that hard after all :D
Color support now autodetected...

File: bin/cmd.js [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 var tapNyan = require('../');
 
+var args = process.argv.slice(2);
```
Comment (calvinmetcalf) [inline bin/cmd.js]: no longer needed

File: README.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ NYAN TAP output ~~inspired by~~ blatantly copied from Mocha's [nyan reporter](ht
 
 ![screen shot](http://i.imgur.com/T1eH156.png)
 
+**ANSI colors**
```
Comment (calvinmetcalf) [inline README.md]: this is no longer up to date, we should probably replace it with a reference to the supported flags in `supports-color`

**Comment by Tokimon** [comment]
Thanks for the catch. I corrected it.

File: README.md (hunk 2 above)
Comment (Tokimon) [inline README.md]: I am not sure wee need to add the `supports-color` flags description. They are not that important imho...

Comment (Tokimon) [inline README.md]: I have added a few word regarding the colors. Check to see if you are ok with it.

**Pull request merged**